server/norace_test.go

```diff
@@ -7154,7 +7154,7 @@ func TestNoRaceJetStreamInterestStreamCheckInterestRaceBug(t *testing.T) {
 
 	numToSend := 10_000
 	for i := 0; i < numToSend; i++ {
-		_, err := js.PublishAsync("foo", nil)
+		_, err := js.PublishAsync("foo", nil, nats.StallWait(800*time.Millisecond))
```

Comment:
other test that fails in a similar way is:

```
=== RUN   TestNoRaceJetStreamSparseConsumers/MemoryStore
    norace_test.go:4325: require no error, but got: nats: stalled with too many outstanding async published messages
    --- FAIL: TestNoRaceJetStreamSparseConsumers/MemoryStore (10.46s)
```

Reply:
Will take a look at that too.